demisto_sdk/commands/content_graph/parsers/parsing_rule.py

```diff
@@ -10,13 +12,21 @@
 
 class ParsingRuleParser(YAMLContentItemParser, content_type=ContentType.PARSING_RULE):
     def __init__(
-        self, path: Path, pack_marketplaces: List[MarketplaceVersions]
+        self,
+        path: Path,
+        pack_marketplaces: List[MarketplaceVersions],
+        git_sha: Optional[str] = None,
     ) -> None:
-        super().__init__(path, pack_marketplaces)
+        super().__init__(path, pack_marketplaces, git_sha=git_sha)
+
+    @cached_property
+    def mapping(self):
+        super().mapping.update({"object_id": "id"})
+        return super().mapping
 
     @property
     def object_id(self) -> Optional[str]:
-        return self.yml_data.get("id")
+        return get(self.yml_data, self.mapping.get("object_id", ""))
```

Comment:
we can remove all those properties in all the subclasses.
The impl is identical to YAMLContentItemParser.